recipe_scrapers/__init__.py

```diff
@@ -605,22 +606,44 @@ def scraper_exists_for(url_path: str) -> bool:
     return host_name in get_supported_urls()
 
 
-def scrape_me(url_path: str, **options: Any) -> AbstractScraper:
-    host_name = get_host_name(url_path)
+HEADERS = {"User-Agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:86.0) Gecko/20100101 Firefox/86.0"}
```

Comment:
Could you move this into a `_network.py` module, and re-use that from both here and `_abstract.py`, to avoid duplicate declaration of the user-agent?

Reply:
modifying `_abstract.py` causes a few issues because there are multiple scrapers that import the `HEADERS` variable directly from `_abstract.py`. However, I should be able to just import the variable directly, which would avoid the duplicate declaration